### PR TITLE
gay

### PR DESCRIPTION
**Pull request by neeasade**

### CODE REVIEW

**Comment by neeasade** [comment]
you might also consider the builtin 'future' font, it's pretty neat.

**Comment by kevindice** [comment]
Yeah, I saw the `-f gay` option.  After messing with it for a while, I managed to get about what I want.  It's kindof a pain that I have to use crontab to output to a file that is used.

**Comment by V-FEXrt** [comment]
Such an odd flag name.

**Comment by neeasade** [comment]
:+1:

**Comment by kevindice** [comment]
Don't do a google image search for `toilet -f gay`

**Comment by neeasade** [comment]
>  It's kindof a pain that I have to use crontab to output to a file that is used.

I mean if you wanted to you could `watch -n 120 /your/script.sh` in a screen. cron is probably better/more permanent, though. 

> Don't do a google image search for toilet -f gay
heh

**Comment by kevindice** [comment]
The issue was that I have `linuxlogo` giving me realtime stats.
banner

**Comment by neeasade** [comment]
Right, I got that. My suggestion is invalid through reboots.

**Comment by V-FEXrt** [comment]
Could you then just have a start up script that activates it on reboot?

**Comment by neeasade** [comment]
> Could you then just have a start up script that activates it on reboot?

Right but at that point cron is a preferrable option for this use case IMO. 

Though it would be easy enough. systemd is comfy.

**Comment by kevindice** [comment]
Yes, but crontab is perfect for this situation.  You want to run something repeatedly on a schedule?  :boom: Crontab!

**Comment by V-FEXrt** [comment]
👍

**Comment by kevindice** [comment]
Here's another set of cron jobs I've got on our server:
```20 2 * * 1 /usr/bin/git -C /opt/letsencrypt pull
30 2 * * 1 /opt/letsencrypt/letsencrypt-auto renew >> /var/log/letsencrypt-renew.log
35 2 * * 1 /bin/systemctl reload nginx
```

Line 1:  Pulls down the letsencrypt repository to get any updates we need at 2:20am every Monday
Line 2:  Runs the auto-renew script from letsencrypt to update any SSL certs that are soon to expire at 2:30am every Monday
Line 3:  Reloads Nginx so any new SSL certs are used at 2:35am every Monday

tl;dr crons make your life easy
